README.md

```diff
@@ -35,7 +35,187 @@ and your application runs on a consistent set of the dependencies.
 
 The content of the Libraries BOM consists of 2 categories:
 - Google Cloud Java client libraries (Maven coordinates `com.google.cloud:google-cloud-XXX`, where XXX is a GCP service name) and
-- core Google dependency libraries, such as gRPC, Protobuf, and Guava.
+- Core Google dependency libraries, such as gRPC, Protobuf, and Guava.
+
+This is the table of modules included in the latest libraries-bom release:
```

Comment:
Is it possible to provide the actual libraries-bom version here?

Reply:
Updated to include!